examples/demo-custom-command/atmos.yaml

```diff
@@ -1,25 +1,5 @@
 base_path: "./"
 
-components:
```

Comment:
removed this to have minimal yaml for demo.
We should not validate any other section while running custom command. So this should be fine. Tests are already present for this, so there was no need to add seperate tests.
I know we generally don't like logs section being deleted. So let me know if want it. 😅 .